blog/2022/02/10years.md

```diff
@@ -0,0 +1,213 @@
+@def rss_pubdate = Date(2021, 2, 14)
+@def rss = """10 years ago today, we published 'Why we Created Julia' ..."""
+@def published = "14 February 2021"
+@def title = "Why We Use Julia, 10 Years Later"
+@def authors = """The Julia Community"""
+@def mintoclevel=2 
+@def maxtoclevel=3
+
++++
+meta = [ ("property", "og:image", "https://user-images.githubusercontent.com/35577566/153918544-6f2fe98e-9019-43ba-ab4a-d922441688cb.png"), ("name", "twitter:creator:id", "194427250"), ("name", "twitter:card", "summary_large_image") ]
++++
+
+~~~
+<img src="https://user-images.githubusercontent.com/35577566/153917389-f9e3cbbf-8c14-47f7-a8ae-a3dcbd60f4a8.png" width="80%" alt="Why We Use Julia, 10 Years Later">
+~~~
+
+
+# Foreword
+
+Exactly ten years ago today, we published "[Why We Created Julia](https://julialang.org/blog/2012/02/why-we-created-julia/)", introducing the Julia project to the world. At this point, we are well on our way to meeting the ambitious goals set out in the original blog post. The incredible community of people that have coalesced around Julia are changing the world — from vaccines to climate modeling, material science, macroeconomics — leading-edge work in every imaginable field of science and technology. Entire companies are being formed that build their software stacks on Julia. Some lines of code in a git repository give an open source project its form, but a community gives it life. We continue to be impressed every day by the breadth of knowledge and kindness of spirit of the the people who have gravitated to Julia. Because of that, we wanted to give this space to them. Below we collate the stories of some of the members of the Julia community (both old and new) in a shared reflection on the past 10 years.
```

Comment:
```suggestion
Exactly ten years ago today, we published "[Why We Created Julia](https://julialang.org/blog/2012/02/why-we-created-julia/)", introducing the Julia project to the world. At this point, we we have moved well past the ambitious goals set out in the original blog post. Julia is now used by hundreds of thousands of people. It is taught in hundreds of universities and entire companies are being formed that build their software stacks on Julia. From vaccines to climate modeling, material science, macroeconomics — everywhere you look, members of the Julia community are pushing the boundary of what's possible.  Some lines of code in a git repository give an open source project its form, but a community gives it life. We continue to be impressed every day by the breadth of knowledge and kindness of spirit of the the people who have gravitated to Julia. Because of that, we wanted to give this space to them. Below we collate the stories of some of the members of the Julia community (both old and new) in a shared reflection on the past 10 years.
```

Otherwise the community idea is broken up, which I don't think reads as well.